itl/plugins-contrib.d/web.conf

```diff
@@ -582,6 +582,10 @@ object CheckCommand "ssl_cert" {
 			set_if = "$ssl_cert_ignore_tls_renegotiation$"
 			description = "Do not check for renegotiation"
 		}
+		"--maximum-validity" = {                                                                                                                                                                                                                                                        
+			value = "$ssl_cert_maximum_validity$"                                                                                                                                                                                                                                       
+			description = "The maximum validity of the certificate must not exceed 'days' (default 397)"                                                                                                                                                                                
+		}                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                       
```

Comment:
As a user reading this I have no idea which _'days'_ are meant. I'd replace

    must not exceed 'days'

with

    in days

Reply:
done. (text had been taken from the script itself)